### PR TITLE
fix(extension-list): fix CJK IME in the list

### LINKED ISSUE #1017: CJK IME doesn't work in a task list and a collapsible list
### Summary

I can't use CJK (Chinese, Japanese, and Korean) IME (input method editor) to input text in a task list and in a collapsible list.

### Steps to reproduce

https://user-images.githubusercontent.com/24715727/126677840-ea9c1441-1b3d-439c-8530-fed8874b8844.mp4

### Expected results

I can input it. 

### Actual results

The IME didn't show up.

### Screenshot(s)

Check the video above.

### PR DESCRIPTION
**Pull request by ocavue**
### Description

Fixes #1017

### Checklist

- [x] I have read the [**contributing**](https://github.com/remirror/remirror/blob/HEAD/docs/contributing.md) document.
- [x] My code follows the code style of this project and `pnpm fix` completed successfully.
- [x] I have updated the documentation where necessary.
- [x] New code is unit tested and all current tests pass when running `pnpm test`.

### Screenshots

https://user-images.githubusercontent.com/24715727/126678591-eddd70db-d11a-4aed-9f24-611e399a98e2.mp4

### CODE REVIEW

File: packages/remirror__extension-list/src/list-item-extension.ts [hunk 1]
```diff
@@ -87,6 +87,7 @@ export class ListItemExtension extends NodeExtension<ListItemOptions> {
       }
 
       return createCustomMarkListItemNodeView({
+        view: this.store.view,
```
Comment (ocavue) [inline packages/remirror__extension-list/src/list-item-extension.ts:90]: There is a small possibility that the `view` here will become `undefined`. I can't reproduce it. Any idea why? @ifiokjr

Comment (ifiokjr) [reply packages/remirror__extension-list/src/list-item-extension.ts:90]: I'm not sure, let's look into it together.

**Review by ifiokjr — APPROVED** [review]
Good stuff

**Comment by ifiokjr** [comment]
I'm happy for you to merge this when you're happy with the changes.

File: packages/remirror__extension-list/src/list-item-extension.ts (hunk 1 above)
Comment (ocavue) [reply packages/remirror__extension-list/src/list-item-extension.ts:90]: OK. Let me create a minimal reproduction in another issue.

**Pull request merged**